packages/react/README.md

```diff
@@ -78,68 +90,78 @@ initializeFaro({
     new ReactIntegration({
       // Only needed if you want to use the React Router instrumentation
       router: {
+        // version: ReactRouterVersion.V6 // => change to .V6_data_router,
         version: ReactRouterVersion.V6_data_router,
         dependencies: {
           matchRoutes,
+          // +++ remove the following dependencies +++
+          // createRoutesFromChildren,
+          // Routes,
+          // useLocation,
+          // useNavigationType,
         },
       },
     }),
   ],
 });
+```
+
+Updating the router instrumentation
 
-// To instrument the router you need to attach Faro instrumentations providing it to the withFaroRouterInstrumentation function
-// Do this in your App.js or other file where you create the router.
+1. Remove `<FaroRoutes>` component. This will not work anymore with V6 data routers.
+2. Create a data router and wrap it with `withFaroRouterInstrumentation(dataRouter)`
 
+Example: Instrument Router
+
+```ts
 const reactBrowserRouter = createBrowserRouter([
-  //...
+  // your routes
 ]);
 
 const browserRouter = withFaroRouterInstrumentation(reactBrowserRouter);
 ```
 
-### Error Boundary
-
-```tsx
-import { FaroErrorBoundary } from '@grafana/faro-react';
-
-// during render
-<FaroErrorBoundary>
-  <App />
-</FaroErrorBoundary>;
-```
-
-or
-
-```tsx
-import { withErrorBoundary } from '@grafana/faro-react';
+## React router without
 
-export default withErrorBoundary(App);
-```
+To set up Faro-React with React Router V5 or V6 without Data routers, add the following code snippet
```

Comment:
There use to be a v4, is that no longer relevant?

Reply:
Good catch tynak you. Forgot to add it.